### ISSUE TITLE
Support 'mask' in coreos.units

### ISSUE THREAD

**Issue by crawford** [coreos/bugs#716]
<a href="https://github.com/robszumski"><img src="https://avatars.githubusercontent.com/u/755540?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [robszumski](https://github.com/robszumski)**
_Thursday Mar 27, 2014 at 18:58 GMT_
_Originally opened as https://github.com/coreos/coreos-cloudinit/issues/54_

----

Supporting `mask` makes it easy to inhibit auto reboots:

```
coreos:
    units:
      - name: update-engine-reboot-manager.service
        command: mask
      - name: update-engine-reboot-manager.service
        command: stop
```

**Comment by crawford** [comment]
<a href="https://github.com/marineam"><img src="https://avatars.githubusercontent.com/u/153640?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [marineam](https://github.com/marineam)**
_Thursday Mar 27, 2014 at 20:24 GMT_

----

Likewise: unmask, also enable/disable may be useful.

**Comment by crawford** [comment]
<a href="https://github.com/jonboulle"><img src="https://avatars.githubusercontent.com/u/2601015?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jonboulle](https://github.com/jonboulle)**
_Saturday May 10, 2014 at 08:16 GMT_

----

I inadvertently implemented this (https://github.com/coreos/coreos-cloudinit/pull/100/files#diff-5c65e1ffe3eb1dda92cb046866646d08R168), but maybe it should look different?

**Comment by crawford** [comment]
<a href="https://github.com/philips"><img src="https://avatars.githubusercontent.com/u/33544?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [philips](https://github.com/philips)**
_Friday May 16, 2014 at 01:49 GMT_

----

@jonboulle should this be closed then?

**Comment by crawford** [comment]
<a href="https://github.com/jonboulle"><img src="https://avatars.githubusercontent.com/u/2601015?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jonboulle](https://github.com/jonboulle)**
_Friday May 16, 2014 at 01:50 GMT_

----

Yeah, if that implementation (a simple boolean at the Unit level, rather than `command: mask` option) is ok for now, this can be closed. @marineam might have an opinion :-)

**Comment by crawford** [comment]
<a href="https://github.com/marineam"><img src="https://avatars.githubusercontent.com/u/153640?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [marineam](https://github.com/marineam)**
_Friday May 16, 2014 at 02:06 GMT_

----

No opinion here, a boolean is fine

**Comment by crawford** [comment]
<a href="https://github.com/jonboulle"><img src="https://avatars.githubusercontent.com/u/2601015?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jonboulle](https://github.com/jonboulle)**
_Friday May 16, 2014 at 02:08 GMT_

----

Fixed by #100
